common/src/test/scala/org/apache/celeborn/common/CelebornConfSuite.scala

```diff
@@ -177,4 +177,11 @@ class CelebornConfSuite extends CelebornFunSuite {
     assert(conf.networkConnectTimeout.duration.toMillis == 2000L)
     assert(conf.networkIoConnectTimeoutMs("data") == 2000L)
   }
+
+  test("CELEBORN-601: Consolidate configsWithAlternatives with `ConfigBuilder.withAlternative`") {
+    val conf = new CelebornConf()
+      .set("rss.network.timeout", "300s")
```

Comment:
`rss.` alias is going to be removed, maybe we can create a dedicated one for testing

Reply:
DOne